task 2/2 intro task.playground/Contents.swift

```diff
@@ -0,0 +1,200 @@
+/**
+Class item for implementation class Stack. Has properties: data (Int) and previousElement (StackElement?)
+ */
+public class StackElement {
+    public var data: Int
+    public var previousElement: StackElement?
+    
+    public init(data: Int, previousElement: StackElement) {
+         self.data = data
+         self.previousElement = previousElement
+    }
+    
+    public init(data: Int) {
+        self.data = data
+    }
+}
+
+/**
+Stack implementation. Based on StackElement entities.
+ */
+public class Stack {
+    private var firstElement: StackElement?
+    private var lastElement: StackElement?
+    
+    public init(){}
+    
+    /**
+     Add element on the top of the stack. Complexity: O(1)
+     - parameter element: Element for push to stack
+     */
+    public func push(_ data: Int) {
+        let newStackElement = StackElement(data: data)
+        if self.firstElement == nil {
+            self.firstElement = newStackElement
+        } else {
+            newStackElement.previousElement = self.lastElement!
+        }
+        self.lastElement = newStackElement
+    }
+    
+    /**
+     Returns the top element of the stack, but does not remove it. Complexity: O(1)
+     - returns: The top element of the stack, but does not remove it.
```

Comment:
Don't add too much details in `returns` description. It doesn't matter if the item wasn't removed. I want to know what is the *return* value.
```suggestion
     - returns: The top element of the stack.
```

Plus you duplicate the information from the description.